src/forms/labels/labeled-field.js

```diff
@@ -20,6 +20,7 @@ import { hasInputError } from '../helpers'
  * @param {Boolean} [hideErrorLabel] - A boolean determining whether to hide the error label on input error (optional, default `false`)
  * @param {Function} [labelComponent=InputLabel] - A custom label component for the input
  * @param {Function} [errorComponent=InputError] - A custom error component for the input
+ * @param {Element Type} [as] - A string that determines the element type of the component (optional, default `div`)
```

Comment:
If this prop was really a string, then the @param should be `String` right? But `propTypes.elementType` supports functional components and class components as well (https://github.com/facebook/prop-types/pull/211/files).

Do you have a source for using `elementType` for `"as"`?